ui/pages/hyperliquid-referral-consent/hyperliquid-referral-consent.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { fireEvent, screen } from '@testing-library/react';
+import configureMockStore from 'redux-mock-store';
+import { renderWithProvider } from '../../../test/lib/render-helpers';
+import mockState from '../../../test/data/mock-state.json';
+import { HyperliquidReferralConsent } from './hyperliquid-referral-consent';
+
+const defaultProps = {
+  onActionComplete: jest.fn(),
+  allAccounts: true,
+  selectedAddress: '0x123',
+};
+
+const mockStore = configureMockStore([]);
+
+describe('HyperliquidReferralConsent', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should render the component with correct title and subtitle', () => {
```

Comment:
Minor, `should` is a little superfluous as it would be on every test, we try and just start with the verb like `renders` or `throws` etc.